app/src/main/res/values-it/strings.xml

```diff
@@ -19,22 +19,47 @@
     <string name="buttonAddFiles">Aggiunta di file</string>
     <string name="buttonNext">Avanti</string>
     <string name="contentDescriptionCreateNewTransferButton">Nuovo trasferimento</string>
+    <string name="downloadedTransferLabel">Trasferimento scaricato : %d/%d</string>
+    <string name="expiresIn">Scade tra %d giorni</string>
```

Comment:
Is there a singular form for this string?
Maybe we will display "Expires tomorrow" instead of "Expires in 1 day"?

Reply:
I asked on figma. I have deleted this string for now on my side since it's not used in this PR.